pkg/operator/ceph/object/user/controller.go

```diff
@@ -705,25 +705,28 @@ func (r *ReconcileObjectStoreUser) objectStoreInitialized(cephObjectStoreUser *c
 	return errors.New("no rgw pod found")
 }
 
-func (r *ReconcileObjectStoreUser) getObjectStore(storeName string) (*cephv1.CephObjectStore, error) {
+func (r *ReconcileObjectStoreUser) getObjectStore(osu *cephv1.CephObjectStoreUser) (*cephv1.CephObjectStore, error) {
```

Comment:
nit: simpler for readability
```suggestion
func (r *ReconcileObjectStoreUser) getObjectStore(user *cephv1.CephObjectStoreUser) (*cephv1.CephObjectStore, error) {
```